src/components/Message.js

```diff
@@ -471,10 +471,10 @@ export class Message extends Component {
         handleDelete={this.handleDelete}
         handleEdit={this.handleEdit}
         handleRetry={this.handleRetry}
-        isMyMessage={this.isMyMessage}
-        openThread={
+        handleOpenThread={
```

Comment:
`openThread` function is also present in ChannelContext. Normally not much of an issue, but if the custom UI Message component is wrapped with channel context, then it creates a conflict in typescript.